db/schema.rb

```diff
@@ -430,7 +430,6 @@
     t.datetime "opening_at"
     t.datetime "closing_at"
     t.integer  "corporate_information_page_type_id",          limit: 4
-    t.string   "need_ids",                                    limit: 255
```

Comment:
I think you need to check the schema back out as the migration that caused this is no longer here.